setup.py

```diff
@@ -25,6 +32,55 @@ def find_version(*file_paths):
 VERSION = find_version('torchtext', '__init__.py')
 long_description = read('README.rst')
 
+
+def get_extensions():
+    this_dir = os.path.dirname(os.path.abspath(__file__))
+    extensions_dir = os.path.join(this_dir, 'torchtext', 'csrc')
+
+    main_file = glob.glob(os.path.join(extensions_dir, 'text_extension.cpp'))
+    source_core = glob.glob(os.path.join(extensions_dir, 'core', '*.cpp'))
+
+    sources = main_file + source_core
+    extension = CppExtension
```

Comment:
can we just use CppExtension instead of creating a variable for it?

Reply:
```extension``` can later be set to ```CUDAExtension``` but right now we have a function that has nothing to do with torch so I think that we should leave it this way.